app/src/main/kotlin/com/tailoredapps/androidapptemplate/ui/overview/OverviewScreen.kt

```diff
@@ -0,0 +1,24 @@
+package com.tailoredapps.androidapptemplate.ui.overview
+
+import androidx.compose.foundation.Image
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import coil.compose.rememberImagePainter
+import com.tailoredapps.androidapptemplate.navigation.Navigator
+import org.koin.androidx.compose.get
+import org.koin.androidx.compose.getViewModel
+
+@Composable
+fun OverviewScreen(
+    navigator: Navigator = get(),
+    viewModel: OverviewViewModel = getViewModel(),
+) {
+    val viewModelState by viewModel.state.collectAsState()
+
+    Image(
+        painter = rememberImagePainter(viewModelState.logoUrl),
+        contentDescription = null,
+    )
+}
```

Comment:
Add a more complex example, showcase a proper separation of "inject-able composables" and "pure composables"

Reply:
What do you mean with "inject-able composables"?

Reply:
imo we should separate between composables which are able to receive dependencies via injecetion (i.e. viewmodel, navigator etc) and pure composables which are resposible for the UI and interaction (through lambdas)